docs/reference/ilm/apis/delete-lifecycle.asciidoc

```diff
@@ -6,15 +6,17 @@
 <titleabbrev>Delete Policy</titleabbrev>
 ++++
 
-Deletes an existing lifecycle policy
+Deletes a lifecycle policy.
 
 ==== Request
 
-`DELETE _ilm/policy/<policy>`
+`DELETE _ilm/<policy_id>`
```

Comment:
I'm not sure this change is intended and might be due to a merge conflict. The API endpoint is now `_ilm/policy/<policy_id>`

Reply:
Fixed. (And also updated PUT/GET to use policy_id to be consistent here & with how we refer to things in the other ref docs.)